frontend/src/bundles/common/components/avatar/styles.module.scss

```
@@ -0,0 +1,25 @@
+.avatar {
+    img {
+        width: 100%;
+        height: 100%;
+    }
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: var(--white-lilac);
+    font-size: 24px;
+    background-color: var(--azure);
+}
+
+.small {
+    width: 40px;
+    height: 40px;
+    border-radius: 10px;
+}
+
+.medium {
+    width: 48px;
```

```suggestion
    width: 48px;
    min-width: 48px;
```
When I tested it locally, avatar was pressed inside

---

about static width - elipsis well working with fixed values and any others lead to problems (and because of ` white-space: nowrap;`), I haven't found any solution yet. I think even that looks acceptable.

---

about avatar, I will add min-width, but avatar It's not my component